### DISCUSSION TITLE
[Gratitude] Thank you, Team Rizu

### DISCUSSION THREAD

**Discussion by carlshan** [TeamRizu/OutFox#236] [01 - General]
This isn't a bug report. I just wanted to say thanks to the entire TeamRizu team for working on modernizing Stepmania. I started my love of dance rhythm games when I first discovered DDR on the PS2 at age 13. The recent COVID quarantines recently got me looking into cardio exercises that can be done at home and I found out about Stepmania.

After playing on Stepmania for a few months, I discovered the OutFox fork and downloaded it instead. My fiancee and I have been playing multiple times a week and it's our main source of working out and having a good time dancing to funky songs.

I wanted to say thanks to the entire team for doing this for free. Is there any way we can support you all (i.e. a Patreon, BuyMeACoffee)?

**Comment by Joehuu** [comment]
> Is there any way we can support you all (i.e. a Patreon, BuyMeACoffee)?

If you look at the site (https://projectmoon.dance/), there is a Patreon icon at the top right (https://www.patreon.com/TeamRizu).

**Comment by carlshan** [comment]
Thanks! I just signed up to become a Patron.

p.s. On the main website, Content -> Music currently links to a broken page.

**Comment by Scraticus** [comment]
Thanks for your support!

We have been told by a lot of our players in the discord server that our 'timing' on going public with the alphas just before the pandemic hit has helped a lot of folks, which has been awesome.

We are continuing to work and thrash out the bugs, so do check back for updates and bugfixes, 4.9.5 is being prepared as we speak.

thanks a lot for your positive comments, it's nice to hear from you!
